src/main/resources/SQLScript.sql

```diff
@@ -0,0 +1,20 @@
+CREATE TABLE _user (
+id VARCHAR(50),
+name VARCHAR(50),
+creation_table VARCHAR(50)
+);
+
+CREATE TABLE ticket(
+id VARCHAR(50),
+user_id VARCHAR(50),
+ticket_type VARCHAR(50),
```

Comment:
create your custom type (`ticket_type`) for db and use it for ticket_type field instead of `varchar(10)`